modules/nw-ovn-kubernetes-migration.adoc

```diff
@@ -226,6 +163,47 @@ The machine config must include the following update to the systemd configuratio
 ExecStart=/usr/local/bin/configure-ovs.sh OVNKubernetes
 ----
 
+. To start the migration, set an annotation on the Cluster Network Operator configuration object by entering the following command:
++
+[source,terminal]
+----
+$ oc annotate Network.operator.openshift.io cluster \
+  'networkoperator.openshift.io/network-migration'="start"
```

Comment:
It seems need --overwrite in this command.

Reply:
@huiran0826, based on what I saw on Slack yesterday, I think this feature might be getting substantially refactored.

Reply:
@jboxman Thanks for the information. Just seeing the slack conversation and the new PR https://github.com/openshift/api/pull/884. Wait for the new procedure finalised.